profiles/tests/test_filter_profiles.py

```diff
@@ -13,7 +13,7 @@ def setUpTestData(cls):
         test_categories = []
 
         # setup activities
-        HEADER_ACTIVITIES = ["producer", "importer", "retail", "HORACE"]
+        HEADER_ACTIVITIES = ["producer", "importer", "retail", "HORECA"]
```

Comment:
I'd recommend to:
1. Use constants instead of 'magic strings'.
2. Be consistent in letters case. If all of the strings in this list are codes, make them either all lowercase or all uppercase, but not mixed.